sourabh/Html_assignment4.txt

```diff
@@ -0,0 +1,10 @@
+Q 4.1 Is there any best practice related to how often an H1 tag should ideally appear in a single web
```

Comment:
you've repeat this question, also why there is a difference in answers of previous and this one